build.sh

```diff
@@ -1,4 +1,4 @@
 rm -rf dist/*
-python3 setup.py sdist bdist_wheel
-twine upload dist/*
-rm -rf dist/ build/ heavyball.egg-info/
\ No newline at end of file
+python -m build
```

Comment:
woah, that's cool

Reply:
You will need to install `build` (`pip install build`). But that is pretty straightforward .